.nvmrc

```diff
@@ -1 +0,0 @@
-18
```

Comment:
I find it useful for projects to have an `.nvmrc` file because I'm using the [nvm auto-switch script](https://github.com/nvm-sh/nvm#zsh). It makes my life easier by switching the Node version automatically the moment I enter the project's dir.

Could we keep the file and maybe just update the version to 20?

Reply:
+1

Reply:
Okay, then I reverted it to the new node version.

I did it because of this comment from here: https://github.com/Automattic/sensei/issues/7722#issuecomment-2622703046